test/fixtures/pre_release_package_info_response.dart

```diff
@@ -0,0 +1,428 @@
+// ignore_for_file: prefer_single_quotes, lines_longer_than_80_chars
+// Generated from: https://pub.dev/api/packages/very_good_cli
```

Comment:
```suggestion
// Generated from: https://pub.dev/api/packages/mason
```
Is this meant to be mason?

Could we also add the date and time of the request to the comment for "simpler" record keeping?
```suggestion
// Generated from: https://pub.dev/api/packages/mason (Accessed: 28th March 2023 at 15:38UTC)
```

Reply:
That would even be accurate since I removed irrelevant versions for the test (otherwise, this pr would have 4k+ lines of change). It is a fixture for the test first, a snapshot second.

Reply:
Oh, I was not aware that this was a modified snapshot of the actual response. I've opened https://github.com/VeryGoodOpenSource/pub_updater/issues/43 to resolve this issue.

Feel free to resolve this conversation.